lively.ide/world.js

```diff
@@ -205,8 +205,11 @@ export class LivelyWorld extends World {
     const activeWindow = this.activeWindow();
 
     if (activeWindow && target === this) activeWindow.deactivate();
-
-    if (!target.isFlap) this.handleHaloCycle(evt);
+    if (
+      this.morphsInWorld.includes(target) || // keep halo and properties panel focus around when interacting with tools
```

Comment:
Maybe I am super confused but isnt the expression doing exactly the opposite of what the comment says?
When `target` is a tool, the expression returns false unlike true as the comment suggests.

Reply:
![many-halos](https://github.com/LivelyKernel/lively.next/assets/1296388/101fde15-7ea4-466a-a8d5-1bf044f07e81)